kotiki-engine/src/kotiki-engine/core/metrics/chebyshev.cpp

```diff
@@ -0,0 +1,12 @@
+#include <algorithm>
+#include <cmath>
+#include "kotiki-engine/core/metrics/chebyshev.hpp"
+
+namespace algo {
+coord_t ChebyshevMetric::Calculate(entity::Entity first, entity::Entity second) {
+    double diff_x = std::abs(first.x - second.x);
```

Comment:
Давай все таки везде использовать явно определенный тип для координат
```suggestion
    coord_t diff_x = std::abs(first.x - second.x);
```